website/src/components/IconButton/IconButton.test.jsx

```diff
@@ -1,24 +1,26 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { shallow, mount, render, configure} from 'enzyme';
+import {
+  shallow,
+  configure,
```

Comment:
why not make it one line?

Reply:
I followed the rule of Eslint on my Aion